src/Configuration.ts

```diff
@@ -49,17 +55,15 @@ export const CONFIG = {
     }
   },
 
-  // BackendApp Configuration
-  // These information are documented here:
-  // https://docs.google.com/document/d/1Qqe6mSfon-blHzc-ldeEHmzIkVaElKY5LtDnKiLbk80/edit
-  // Used to communicate with Backend App
-  BACKEND_APP: {
-    HOST: process.env.BACKEND_APP_HOST || localhost,
-    PORT: process.env.BACKEND_APP_PORT || 3002,
-    SERVICES: {
-      PAYMENTS_STATUS_UPDATE: "/payment/status/update"
-    }
-  }
+  // Redis DB Server Configuration
+  REDIS_DB: {
+    PORT: process.env.REDIS_DB_PORT || 6379,
+    HOST: process.env.REDIS_DB_HOST || "localhost"
+  },
+
+  // Timeout used to store PaymentId into redis db (AttivaRPT process)
+  // #158387557 The value is an estimation that could be reviewed with real scenarios
+  PAYMENT_ACTIVATION_STATUS_TIMEOUT: 60 * 60 * 48
```

Comment:
put the unit of measure into the variable name ie:
PAYMENT_ACTIVATION_STATUS_TIMEOUT_SECONDS 
(is it in seconds ?)